src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml

```diff
@@ -33,8 +33,14 @@
 			<TransitionCollection />
 		</controls:BladeView.ItemContainerTransitions>
 
-		<controls:BladeItem>
-			<Frame x:Name="MainPageFrame" />
+		<controls:BladeItem
+			ManipulationDelta="BladeItem_ManipulationDelta"
+			ManipulationMode="TranslateX"
+			ManipulationStarted="BladeItem_ManipulationStarted"
+			PointerExited="BladeItem_PointerExited"
+			PointerMoved="BladeItem_PointerMoved">
```

Comment:
I think similar to the SideBar, having a separate element would give us more flexibility in the long term. Accessibilitywise, if we also want to handle arrow keys, a separate element would also be better since you can have a dedicated UIA name for the resizer element.

Reply:
That's what I was thinking 👍

Reply:
@Mohammed-619